duckduckgo/Android
# Feature/remove fire animation

Task/Issue URL: https://app.asana.com/0/488551667048375/818602866600272
Tech Design URL: 
CC: 

**Description**:
Removes the much-unloved 🔥animation to minimise the time required to clear data and get the user back using the app.

**Steps to test this PR**:
1. Perform a 🔥to clear data; verify data is still cleared as expected
1. Note the changed UX; verify the animation is gone and instead there is a momentary black screen before the `BrowserActivity` reloads.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
